cirq-core/cirq/protocols/json_serialization.py

```diff
@@ -819,6 +838,28 @@ def read_json_gzip(
     gzip_raw: Optional[bytes] = None,
     resolvers: Optional[Sequence[JsonResolver]] = None,
 ):
+    """Read a gzipped JSON file that optionally contains cirq objects.
+
+    Args:
+        file_or_fn: A filename (if a string or `pathlib.Path`) to read from, or
+            an IO object (such as a file or buffer) to read from, or `None` to
+            indicate that `json_text` argument should be used. Defaults to
```

Comment:
think this should be `gzip_raw` and not `json_text`

Reply:
Done.